actionview/lib/action_view/unbound_template.rb

```diff
@@ -14,10 +14,26 @@ def initialize(source, identifier, details:, virtual_path:)
       @virtual_path = virtual_path
 
       @templates = Concurrent::Map.new(initial_capacity: 2)
```

Comment:
✂️? Looks like we define `@templates` as a hash a couple lines down now.

Reply:
😳 Thanks for catching. I think we'll want the `Concurrent::Map` just in case of non-MRI having issues with concurrent read/write.